### DISCUSSION TITLE
Optional chaining

### DISCUSSION THREAD

**Discussion by milosjovac** [josdejong/mathjs#2882] [Ideas]
Hello,

Current expression parser doesn't support [Optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) (`a?.b?.c`). Using the `?` operator atm is causing the Syntax error.

It would be very handy to have this available when working with optional field objects.

**Comment by josdejong** [comment]
Support for optional chaining would be great! Thanks for your suggestion.

If anyone wants to implement this up please drop a message.
